Protokolle/protokoll-13-05.md

```diff
@@ -0,0 +1,143 @@
+   - arbeiten mit feature branches 
+   - direkter push auf den master branch ist ein no go --> ggf. wird master vor direkt pushes geschützt --> nur via pull requests (PR) möglich
+   - commit message convention
+   - Jede Arbeit wird als Issue getrackt
+   - scope und abschluss kriterien
+   - je nach Aufwand des Issues werden "T-Shirt-Größen" je Issue vergeben --> je größer das Shirt, desto größer der Aufwand in Stunden
+   - neue bounty immer mit Readme anlegen und zum bounty board hinzufügen
+   - Bearbeiter schließen Issue mit pull request ab -> review anfragen
+   - Review von jemanden  der Issue nicht bearbeitet hat  
+   - **Next Steps** 
+        - code of conduct hinzufügen, der dem Berlin CoC folgt
+        - Wiki Organisation --> Wie kann History getrakct werden? // Wie könnte eine inhaltlich Struktur aussehen? (pull requests nicht )
+        - Discord-Channel anlegen https://discord.com/invite/RaN4PZ
+
+> Nice to know: Schätzung des Issue-Aufwands basiert auf Fibonacci-Reihe
+
+**Fragen/Anmerkungen:**
+    - wie kann das Wiki umorganisert werden, damit die Inhalte direkter zugreifbar sind?
+    - Wir haben eine Sketch Lizenz, wenn Interesse besteht.
+    - Wie werden in dem Board die unterschiedlichen Aktivitäten der einzelnen Teams organisiert?
+    - Soll der Master branch protected werden?
+    - Warum werden Issues (Bounties) zeitlich so geschätzt, wie sie geschätzt werden?
+    - Sollen Reviewer definiert werden?
+
+
+####Team: Github Pages (Di Wang, Tung Duong, Oussama Bouanani, Omer Hod)
+   - Nutzung von React (JavaScript Framework)
+   - Programmierung mit Javascript anstatt Type Script
+   - Facebooks create-app
+   - neue branch == "project-website"
+   - für deploy: Nutzung von npm (node package manager) scripts
+   - https://fub-hcc.github.io/20-SWP-CodingOpenness/
+   - **Next Steps:** 
+        - Anlegen der Templates für:
+        - Vision des Projektes
+        - beschreibung des Projektkontexts
+        - Teammitglieder
+        - andere content-sites
+
+
+
+####Team: Technik (Julius Brose, Johannes, Pascal, Dennis, Long Dang)
+   - im Issue sind Informationen zu weiterführenden Kursen hinterlegt
+   - Kotlin und Java Kurse
+   - bevorzugte IDE: [android studio](https://developer.android.com/studio/intro)
+   - https://www.thecrazyprogrammer.com/learn-android-programming
+   - [Kotlin Koans](https://play.kotlinlang.org/koans/overview)
+   - **Next Steps:**
+        - Essenz der Diskussion (im Issue) auf eine Page packen
+
+
+####Team: Vergleich der Ansätze (Claas Fandre, Julia Zimmermann, Linus Helfmann, Bernd Sahre, Viktoriya Kraleva)
+   - Wiki-Page angelegt : https://github.com/FUB-HCC/20-SWP-CodingOpenness/wiki/Vergleich-der-Ans%C3%A4tze
+   - Unterteiwlung aller Kriterien in 3 große Themenbereiche:
+   - Vergleich nach Architektur
+   - zentral / dezantral
+   - ort der Speicherung der Ids
+   - Ort der Berechnung
+   - Verfahren zur Aktualisierung
+   - vergleich Kommunikationsablauf:
+   - Push/ poll
```

Comment:
```suggestion
   - Push/ pull
```

Reply:
poll ist in diesem Fall richtig, steht jedenfalls so im wiki